File: projects/markx-robot/resources/libs/mongo.py
from robot.api.deco import keyword
from pymongo import MongoClient

client = MongoClient('mongodb+srv://douglastony1235_db_user:WCt5ki1cykuuCwGT@cluster0.dwibwni.mongodb.net/markX?appName=Cluster0')

db = client['markX']

@keyword('Remove task from database')
def remove_task_by_name(task_name):
    collection = db['tasks']
    collection.delete_many({'text': task_name})
    print('Removendo a tarefa ' + task_name)

File: projects-javascript/markx-robot/resources/libs/mongo.js
const { MongoClient } = require('mongodb');

const uri = 'mongodb+srv://douglastony1235_db_user:WCt5ki1cykuuCwGT@cluster0.dwibwni.mongodb.net/markX?appName=Cluster0';

async function removeTaskFromDatabase(taskName) {
    const client = new MongoClient(uri);
    try {
        await client.connect();
        const db = client.db('markX');
        const collection = db.collection('tasks');
        await collection.deleteMany({ text: taskName });
        console.log('Removendo a tarefa ' + taskName);
    } finally {
        await client.close();
    }
}

module.exports = {
    removeTaskFromDatabase
};
